relay-general/src/store/normalize.rs

```diff
@@ -814,6 +816,15 @@ pub fn light_normalize_event(
         ); // Measurements are part of the metric extraction
         normalize_breakdowns(event, config.breakdowns_config); // Breakdowns are part of the metric extraction too
 
+        if config.light_normalize_spans {
+            // XXX(iker): span normalization runs in the store processor, but
+            // the exclusive time is required for span metrics. Most of
+            // transactions don't have many spans, but if this is no longer the
+            // case and we roll this flag out for most projects, we may want to
+            // reconsider this approach.
+            spans::normalize_spans(event, &BTreeSet::from([SpanAttribute::ExclusiveTime]));
```

Comment:
Should we also check that the event is a transaction, and the span attribute was actually requested in project configs?

See

https://github.com/getsentry/relay/blob/9dc2bcf145a598de7146f53df58dbf875a27a158/relay-general/src/store/normalize.rs#L197-L199

Reply:
Yes, but I don't want to rely on that flag either (even if it's very unlikely to get changed).